From d7050fa7b0b0e88217ebfc09666eb08632dfe436 Mon Sep 17 00:00:00 2001
From: Laurentiu Tudor <laurentiu.tudor@nxp.com>
Date: Tue, 12 Mar 2019 17:57:26 +0200
Subject: [PATCH] fsl_qbman: add dummy functions for probe deferal APIs

Add missing definitions for probe deferal APIs for drivers
that use them with the SDK version of the qbman drivers.

Signed-off-by: Laurentiu Tudor <laurentiu.tudor@nxp.com>
Signed-off-by: Vlad Pelin <vlad.pelin@nxp.com>
Acked-by: Madalin Bucur <madalin.bucur@nxp.com>
---
 include/linux/fsl_qman.h | 10 ++++++++++
 1 file changed, 10 insertions(+)

--- a/include/linux/fsl_qman.h
+++ b/include/linux/fsl_qman.h
@@ -3893,6 +3893,16 @@ int qman_p_enqueue_orp(struct qman_porta
 int qman_p_enqueue_precommit(struct qman_portal *p, struct qman_fq *fq,
 				const struct qm_fd *fd, u32 flags,
 				qman_cb_precommit cb, void *cb_arg);
+
+static inline int qman_is_probed(void) {
+	return 1;
+}
+
+
+static inline int qman_portals_probed(void) {
+	return 1;
+}
+
 #ifdef __cplusplus
 }
 #endif
